.github/workflows/hpu-gaudi2.yml

```diff
@@ -38,6 +91,13 @@ jobs:
           python -c "import torch; print('torch:', torch.__version__, torch)"
           python -c "import torch; print('CUDA available:', torch.cuda.is_available())"
 
+      - name: Install transformers
+        run: |
+          git clone https://github.com/huggingface/transformers
+          cd transformers
+          git rev-parse --short HEAD
+          pip install .
+
       - name: Install deepspeed
         run: |
           pip install .[dev]
```

Comment:
Should just need to add autotuning here to get the remaining tests to pass

Reply:
Thanks @loadams ,

Updated PR.